lxdock/provisioners/ansible.py

```diff
@@ -14,6 +14,17 @@ class AnsibleProvisioner(Provisioner):
     """ Allows to perform provisioning operations using Ansible. """
 
     name = 'ansible'
+
+    guest_required_packages_alpine = ['openssh', 'python', ]
+    guest_required_packages_archlinux = ['openssh', 'python2', ]
+    guest_required_packages_centos = ['openssh-server', 'python', ]
+    guest_required_packages_debian = ['openssh-server', 'python', ]
```

Comment:
Technically, it's enough, but without `python-apt` and `aptitude`, we can't run the `apt` action, which is used pretty much all the time: http://docs.ansible.com/ansible/apt_module.html#requirements-on-host-that-executes-module